components/Meta/shape.js

```diff
@@ -1,10 +1,10 @@
 import PropTypes from 'prop-types'
-import prismicImageShare from 'shapes/prismic/image'
+import prismicImageShape from 'shapes/prismic/image'
 
 const metaProps = {
   meta_title: PropTypes.string,
   meta_description: PropTypes.string,
-  meta_image: prismicImageShare,
+  meta_image: PropTypes.oneOfType[(PropTypes.string, prismicImageShape)],
```

Comment:
@suhmantha1 are you handling the possibility that the meta_image is a string? It seems like meta_iamge is always checking for a url property.

Reply:
the meta_image set in the defaults (code-based) is a string